src/pages/_app.tsx

```diff
@@ -40,10 +40,14 @@ export default function App({ Component, pageProps }: AppPropsWithLayout) {
   const router = useRouter();
   const authStore = useAuthStore();
   const componentSrc = router.query;
+  const isSignedIn = useMemo(() => authStore.signedIn, [authStore.signedIn]);
```

Comment:
Could probably simplify this and achieve the same affect:

```
const accountId = useAuthStore((store) => store.accountId);
const signedIn = useAuthStore((store) => store.signedIn);
```

And then you could delete this:

```
const authStore = useAuthStore();
```

Pulling in the entire store like that will force the component to re-render when anything in the store changes.